backend/routes/users.js

```diff
@@ -10,149 +10,301 @@
         FOREIGN KEY (photoId) REFERENCES photos(photoId)
```

Comment:
Sooooo the jwt isnt in here, we may have to hold off on putting this in main until Rebecca and Trevor fix JWT issues. Once they fix it, you can pull the fixes in to here and push it. I'm worried that merging this into main in this file will overwrite their JWT stuff.

Reply:
Fixed while resolving merge conflicts